tools/purge_ems.rb

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env ruby
+require File.expand_path('../config/environment', __dir__)
```

Comment:
Not sure how to make this portable to `/tmp` or somewhere.  I guess we can attempt to `Dir.chdir` to APP_ROOT via env var in pods (don't see this env var on appliances), or hardcoded /var/www/miq/vmdb, or detect if we can find this file.

Reply:
Ok, this is done in latest.  You either run from tools directory with `rails runner` or `ruby` or you run from a different directory using `rails runner`.  That should handle both containers/appliances before and after this script is available in the `tools` directory.